daxa/mission/base.py

```diff
@@ -562,6 +569,15 @@ def filter_on_obs_ids(self, allowed_obs_ids: Union[str, List[str]]):
         # Makes sure that the allowed_obs_ids variable is iterable over ObsIDs, even if just a single ObsID was passed
         if not isinstance(allowed_obs_ids, list):
             allowed_obs_ids = [allowed_obs_ids]
+        
+        # Accounting for the wrong ObsID being written on the eROSITA website
+        if '700195' in allowed_obs_ids:
+            allowed_obs_ids.remove('700195')
```

Comment:
Would you like me to add in that I've overwritten this from base.py in the docstring? I thought because its a user facing method its best in the comments